### ISSUE TITLE
Question about dataset of CroPA 

### ISSUE THREAD

**Issue by ranck626** [Haochen-Luo/CroPA#5]
Thank you for your work, it is very inspiring. I'm trying to reproduce your experiment but I'm having some trouble and would like your pointers.

1, How to get a subset of MSCOCO validation set.
I can't find the image dataset in the data folder. After reading the code, I found that the paths /train2014, and /val2014 are needed.
I didn't understand the author's statement "The dataset used in the paper is a subset of MSCOCO validation set. we provide the dataset in the data/ directory."

Thank you for your attention and I hope to hear from you soon.

**Comment by ranck626** [comment]
What counts across different prompts and images？？
“When tested across different prompts and images, the ASRs are all near 0. The ASRs over the models with significantly different architecture are near 0.“”

**Comment by Haochen-Luo** [comment]
The full train and validation datasets can be downloaded from the official MSCOCO website.
